index.js

```diff
@@ -0,0 +1,52 @@
+let button = document.getElementsByClassName("btn btn-danger")[0];
+let result = document.getElementById("result");
+let error = document.getElementById("error");
+
+function isNumber(value) {
+  return !isNaN(+value);
+}
+
+const formulas = {
+  addition: (a, b) => a + b,
+  subtraction: (a, b) => a - b,
+  multiplication: (a, b) => a * b,
+  division: (a, b) => a / b
+};
+
+const calculate = function() {
+  const inputs = [
+    "firstOperand",
+    "secondOperand",
+    "thirdOperand",
+    "fourthOperand"
+  ].map(id => +document.getElementById(id).value);
+  console.log("inputs", inputs);
+
+  if (inputs.filter(id => isNumber(id)).length !== inputs.length) {
+    error.textContent = "Please complete all the fields with numbers!";
+    return (result.textContent = "");
+  }
+
+  const operations = [];
+  document
+    .getElementsByName("operations")
+    .forEach(x => operations.push(x[x.selectedIndex].value));
```

Comment:
can you rename the variable x into something more clear? like: selectElement, operation, selectedOperation

Reply:
![1](https://user-images.githubusercontent.com/50665097/79049331-ac0a1000-7c2b-11ea-8666-e756d9adeb45.JPG)